configure.ac

```diff
@@ -1286,7 +1286,7 @@ else
     CLIENTGUIFLAGS="${CLIENTGUIFLAGS} -DNDEBUG"
 fi
 
-CLIENTGUILIBS="${WX_LIBS}"
+CLIENTGUILIBS="${WX_LIBS} -lX11"
```

Comment:
Why this is needed here?

Reply:
Because it missing a flags.

Reply:
Why this was not needed before? For me it looks like some detection is failing to detect X11 and add this flag automatically.

Reply:
It not fail to detect anything. The flags is coming from `wx-config`. In the source code version when they don't compile with cmake, the order of the flag is different from the order of the flags that generate when compile in vcpkg. that why I need to add this extra flag.

Reply:
Then this should be fixed on wxwidgets port side

Reply:
I removed this flag.